### PR TITLE
Added /handwired/asesmero/6x6

### PR DESCRIPTION
**Pull request by asesmero**
Asesmero 6x6
A simple Otrho 6x6 keyboard for shortcut launch.

"I'm still learning, I hope this pull request is correct."

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/asesmero/6x6/info.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    "manufacturer": "asesmero",
+    "keyboard_name": "6x6 promicro",
+    "maintainer": "asesmero",
+	"url": "https://github.com/asesmero"
+    "development_board": "promicro",
+	
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0066",
+        "vid": "0x0282"
```
Comment (drashna) [inline keyboards/handwired/asesmero/6x6/info.json:11]: This makes me uncomfortable.  While yes, it's not assigned officially...

File: keyboards/handwired/asesmero/6x6/readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Asesmero 6x6
+
+! Asesmero 6x6(https://bit.ly/3mM9xPz)
```
Comment (drashna) [inline keyboards/handwired/asesmero/6x6/readme.md:3]: This should be rehosted on Imgur or the like (if imgur, make sure it's submitted by an account)

File: keyboards/handwired/asesmero/6x6/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[LAYER_0] = LAYOUT_ortho_6x6(
+		MEH(KC_A), MEH(KC_B), MEH(KC_C), MEH(KC_D), MEH(KC_E), MEH(KC_F), 
+		MEH(KC_G), MEH(KC_H), MEH(KC_I), MEH(KC_J), MEH(KC_K), MEH(KC_L), 
+		MEH(KC_M), MEH(KC_N), MEH(KC_O), MEH(KC_P), MEH(KC_Q), MEH(KC_R), 
+		MEH(KC_S), MEH(KC_T), MEH(KC_U), MEH(KC_V), MEH(KC_W), MEH(KC_X), 
+		MEH(KC_Y), MEH(KC_Z), MEH(KC_1), MEH(KC_2), MEH(KC_3), MEH(KC_4), 
+		MEH(KC_5), MEH(KC_6), MEH(KC_7), MEH(KC_8), MEH(KC_9), MEH(KC_0)),
```
Comment (drashna) [inline keyboards/handwired/asesmero/6x6/keymaps/default/keymap.c:32]: Just a nit: 
```suggestion
		MEH(KC_5), MEH(KC_6), MEH(KC_7), MEH(KC_8), MEH(KC_9), MEH(KC_0)
	)
```

File: keyboards/handwired/asesmero/6x6/info.json (hunk 1 above)
Comment (drashna) [reply keyboards/handwired/asesmero/6x6/info.json:11]: In general, I prefer VIDs that are much higher in number.  Past 0x4000 is good, and the further past, the better.   For instance, `0x5A30`, which was generated with this command: 
```sh
echo -n 'asesmero' | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
```

**Review by asesmero — COMMENTED** [review]
Changed vid

File: keyboards/handwired/asesmero/6x6/readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Asesmero 6x6
+
+! Asesmero 6x6(https://i.imgur.com/wG2Lpggh.jpg)
+
+A simple 6x6 Keyboard made for use it as a streamdeck.
+
+
+* Keyboard Maintainer: [Antonio Sesmero](https://github.com/asesmero)
+* Hardware Supported: Arduino ProMicro (ATMEGA32U4)
```
Comment (drashna) [inline keyboards/handwired/asesmero/6x6/readme.md:9]: There is no Arduino Pro Micro.  it's a SparkFun design. 
```suggestion
* Hardware Supported: SparkFun Pro Micro (ATMEGA32U4)
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**